### DISCUSSION TITLE
ImportError: DLL load failed while importing _ext: The specified module could not be found.

### DISCUSSION THREAD

**Discussion by GeorgeCzy** [open-mmlab/mmcv#3183] [Q&A]
# Background
I'm using a win-10 computer to install mmcv. After trying installing with the common "pip" command, the "pre-build package" problem occured. Then I followed the instrctions on [here](https://mmcv.readthedocs.io/en/2.x/get_started/build.html) to install mmcv from source.
# Info
My computer has cuda version of 12.6 and torch version of 2.3.0+cu118. I thought i followed the instructions accurately and successfully in the previous website. However, when checking the mmcv according to the command "python .dev_scripts/check_installation.py", it raised the error:
![image](https://github.com/user-attachments/assets/580f426b-3a06-4dfb-9e20-ae2e8196912f)
I tried restart the computer but nothing changed.
# P.s
It's my first time to raise a question like this, so if there is any information that i forgot to provide just let me know. Thanks a lot! 
(Btw, i'm a sophomore with little experience dealing with problem like this /whimper)

**Comment by vchigrinsky** [comment]
I've got the same problem at linux

There are a lot of imports of _ext module throughout the repository but there is ImportError:

<img width="775" alt="image" src="https://github.com/user-attachments/assets/c54351d0-da44-4ebd-9386-854dbfce41c7" />

**Comment by saracarles** [comment]
Hello, 
I am trying to work with mmcv too and i get this same ModuleNotFoundError: No module named 'mmcv._ext' error. I have been roaming arround the issues for a while and I have tried some of the solutions said but no luck. I am building mmcv in cuda 11.8 and then I have also tried in another computer with cuda 12.9 but no luck anywhere. Does anyone have any new ideas of solutions for this?
Thanks!
